spec/polyn/utils/hash_spec.rb

```diff
@@ -44,6 +44,22 @@
         },
       })
     end
+
+    it "should deep stringify keys in lists" do
+      hash = { a: { b: [{ c: 1 }, { d: 2 }] } }
+      expect(described_class.deep_stringify_keys(hash)).to eq({
+        "a" => {
+          "b" => [
+            { "c" => 1 },
+            { "d" => 2 },
+          ],
+        },
+      })
+    end
+
+    it "should return if not a hash" do
```

Comment:
It looked like there was also an array path, but I don't see a test for that path.

Reply:
ah, I tested if it could handle an array inside, but didn't do one for a top level array. I'll add that